django/db/migrations/autodetector.py

```diff
@@ -436,6 +489,42 @@ def generate_renamed_models(self):
                             self.old_model_keys.append((app_label, model_name))
                             break
 
+    def m2m_field_compare(self, through, rem_through):
+        return (
+            through["from"] == rem_through["from"]
+            and through["to"] == rem_through["to"]
+            and through["field"] == rem_through["field"]
+        )
+
+    def generate_renamed_through_models(self):
+        for app_label, model_name in sorted(self.new_through_models):
```

Comment:
`for (app_label, model_name), through in sorted(self.new_through_models.items()):` saves you the next line and is still readable.